packages/nimble-components/src/theme-provider/tests/tokens.stories.ts

```diff
@@ -19,6 +20,11 @@ import {
     groupHeaderFontColor,
     groupHeaderTextTransform
 } from '../design-tokens';
+import { processUpdates } from '../../testing/async-helpers';
+
+// Setting token default values is done as part of the FAST render queue so it needs to be flushed before reading them
+// https://github.com/microsoft/fast/blob/bbf4e532cf9263727ef1bd8afbc30d79d1104c03/packages/web-components/fast-foundation/src/design-token/custom-property-manager.ts#LL154C3-L154C3
+processUpdates();
```

Comment:
This synchronous emptying of the event queue seems sketchy to me since this is really more like application runtime than synchronous running test code.

I wonder if moving the registerRoot to design-tokens eliminates the need for this altogether.

If not it seems like we should instead use something like a stroybook loader to wait for the tokens to be defined: https://storybook.js.org/docs/html/writing-stories/loaders

Reply:
I changed to use a storybook loader that awaits `waitForUpdateAsync()`. I like that this is a little less intrusive; rather than forcing the design system to process when it otherwise wouldn't, it waits for it to finish on its natural cadence.

Their syntax is bit complex and I couldn't find an approach that satisfied all of our lint rules so I ended up having to pick a syntax and disable a rule. If you want to go deep on it we can but I was starting to hit diminishing returns on investment so left it with a disabled rule.